src/recordlinker/schemas/pii.py

```diff
@@ -25,21 +26,57 @@ class Feature(enum.Enum):
     CITY = "CITY"
     STATE = "STATE"
     ZIP = "ZIP"
-    SSN = "SSN"
     RACE = "RACE"
     GENDER = "GENDER"
     TELECOM = "TELECOM"
     PHONE = "PHONE"
     EMAIL = "EMAIL"
     SUFFIX = "SUFFIX"
     COUNTY = "COUNTY"
-    DRIVERS_LICENSE = "DRIVERS_LICENSE"
+    IDENTIFIER = "IDENTIFIER"
 
     def __str__(self):
         """
         Return the value of the enum as a string.
         """
-        return self.value
+        return self.value    
+class Feature(pydantic.BaseModel):
+    """
+    The schema for a feature.
+    """
+
+    model_config = pydantic.ConfigDict(extra="allow")
+
+    suffix: typing.Optional[IdentifierType] = None
+    attribute: FeatureAttribute
+
+    @classmethod
+    def parse(cls, feature_string: str) -> "Feature":
+        """
+        Parse a feature string in the format 'FEATURE_ATTRIBUTE:SUFFIX' into a Feature object.
+
+        Args:
+            feature_string (str): The string to parse.
+
+        Returns:
+            Feature: A Feature object with attribute and suffix populated.
+        """
+        # Split the feature string on ":"
+        parts = feature_string.split(":", 1)
+
+        if len(parts) == 1:
+            # If no suffix is provided, set only the attribute
+            attribute = FeatureAttribute(parts[0])
+            return cls(attribute=attribute)
+
+        # If a suffix is provided, parse both parts
+        attribute = FeatureAttribute(parts[0])
```

Comment:
What happens if the user specifies "FIRST_NAME:DL"?

Reply:
Just updated to account for this by raising a value error